### DISCUSSION TITLE
(Resuelto) Ya no me sale la mission banckend de JAVASCRIPT

### DISCUSSION THREAD

**Discussion by richirrim** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#583] [General]
A alguien más le pasó algo similar ? Desde el lunes ya tenía habilitado el repo  y hoy que quice entrar ya no estaba, desapareció de la nada.

**Comment by Ricardo-g-garcia** [comment]
si me paso lo mismo, no aparece la misión Javascript

**Reply by richirrim** [reply]
La solución es ir al Microsoft Teams y una vez ahí, te vas a la Mission Backend. Luego a la sección de general, luego en el menu de arriba dice una opción "Invite a Github" clic  ahí y ya después de hacer lo que te pide y obtendrás el acceso de nuevo.

**Comment by Drako9159** [comment]
Me pasa igual, solo entré el lunes para hacer mi blog, pero ya me da error la página.

**Reply by richirrim** [reply]
La solución es ir al Microsoft Teams y una vez ahí, te vas a la Mission Backend. Luego a la sección de general, luego en el menu de arriba dice una opción "Invite a Github" clic  ahí y ya después de hacer lo que te pide y obtendrás el acceso de nuevo.

**Comment by Ricardo-g-garcia** [comment]
Entre de nuevo al link para asignar la misión de Backend y al parecer había puesto mi perfil de GitHub en ves de el repositorio, me mando mensaje que había sido asignado a Misión NodeJs y ya pude acceder al Repositorio de GitHub de NodeJs y también apareció en Teams la Misión
